### PR TITLE
Migrate from cljx to cljc with reader conditinals

### PR DESCRIPTION
**Pull request by myguidingstar**

### CODE REVIEW

File: examples/osm/src/core.cljs [hunk 1]
```diff
@@ -1,59 +0,0 @@
-(ns examples.osm.core
```
Comment (pld) [inline examples/osm/src/core.cljs]: Why delete the examples?

File: project.clj [hunk 2]
```diff
@@ -25,53 +23,13 @@
...
-  :prep-tasks [["cljx" "once"] "javac" "compile"]
-  :cljx {:builds [{:source-paths ["src/cljx"]
-                   :output-path "target/generated/src/clj"
-                   :rules :clj}
-                  {:source-paths ["src/cljx"]
-                   :output-path "target/generated/src/cljs"
-                   :rules :cljs}]}
-  :cljsbuild {
-    :builds [{:id "stolen"
-              :source-paths ["src/cljs"
```
Comment (pld) [inline project.clj]: Why remove the bulls for the examples?

Comment (pld) [inline project.clj]: Builds

Comment (myguidingstar) [inline project.clj]: I got confirmed from @prabhasp He moved them to gh-pages already but hasn't cleaned up. This is the clean up

Comment (myguidingstar) [inline project.clj]: they made quite annoying compilation errors btw :)

**Comment by pld** [comment]
:+1: looks great

**Comment by prabhasp** [comment]
+1 thanks @thang!
Should we release 0.1.4? If so, the two changes to add to changelog.md are
* filter added for categorical view bys
* cljx converted to cljc

From my phone,
Prabhas

**Comment by pld** [comment]
Yes, I think this is big enough to warrant a release, apologies for not
doing this immediately after merge, it's deployed on 1.4-SNAPSHOT now

**Pull request merged**